core/platform/cloud_translate/cloud_translate_emulator_test.py

```diff
@@ -0,0 +1,55 @@
+# coding: utf-8
+#
+# Copyright 2021 The Oppia Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS-IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Tests for cloud_translate_emulator."""
+
+from __future__ import absolute_import  # pylint: disable=import-only-modules
+from __future__ import unicode_literals  # pylint: disable=import-only-modules
+
+from core.platform.cloud_translate import cloud_translate_emulator
+from core.tests import test_utils
+
+
+class CloudTranslateEmulatorUnitTests(test_utils.TestBase):
+    """Tests for cloud_translate_emulator."""
+
+    def setUp(self):
+        super(CloudTranslateEmulatorUnitTests, self).setUp()
+        self.emulator = cloud_translate_emulator.TranslateEmulator()
+
+    def test_init_prepopulates_responses(self):
+        self.assertDictEqual(
```

Comment:
Please just use assertEqual, it'll choose the right thing.

Reply:
Fixed.